internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -1949,6 +1949,22 @@ func haltIfInstalled(seq runtime.Sequence, _ any) (runtime.TaskExecutionFunc, st
 	}, "haltIfInstalled"
 }
 
+// MountTmpFilesystems represents the MountTmpFilesystems task.
+func MountTmpFilesystems(runtime.Sequence, any) (runtime.TaskExecutionFunc, string) {
+	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) error {
+		tmpPoints := mountv2.Points{
+			mountv2.NewPoint("tmpfs", "/run", "tmpfs", mountv2.WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_RELATIME), mountv2.WithData("mode=0755")),
+			mountv2.NewPoint("tmpfs", "/system", "tmpfs", mountv2.WithData("mode=0755")),
+			mountv2.NewPoint("tmpfs", "/tmp", "tmpfs", mountv2.WithFlags(unix.MS_NOSUID|unix.MS_NOEXEC|unix.MS_NODEV), mountv2.WithData("size=64M"), mountv2.WithData("mode=0755")),
```

Comment:
We do not actually need to move /tmp, as it's not labeled (uses tmpfs_t gained by its type). Please share your thoughts whether or not we should mount it here or in initramfs